README.md

```diff
@@ -308,7 +308,7 @@ If you want to revise the default configuration for kube-prometheus, for example
 
 #### Kubernetes Deployment
 
-To deploy this project, you can simply run `kubectl apply -f examples/standard` and a Kubernetes service and deployment will be created. (Note: Adjust the apiVersion of some resource if your kubernetes cluster's version is not 1.8+, check the yaml file for more information).
+To deploy this project, you can simply run `kubectl apply -k github.com/kubernetes/kube-state-metrics/examples/standard` and a Kubernetes service and deployment will be created. (Note: Adjust the apiVersion of some resource if your kubernetes cluster's version is not 1.8+, check the yaml file for more information).
```

Comment:
I have a slight preference to keep the default deployment without kustomize on the doc. Could we keep the `kubectl apply -f` here or will this fail if the kustomization file is there?

Reply:
[Kustomization](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/) is considered one standard tool of kubectl already.
I prefer kustomization for the both remote/local install option. You can also install locally with `kubectl apply -k standard`
Otherwise, we imply to the users that they have to clone the project before installation. 


The `kubectl apply -f` will not fall but will generate an error when trying to apply the `kustomization.yaml` to cluster. (other files are applied correctly)
```bash
$ kubectl apply -f standard
clusterrolebinding.rbac.authorization.k8s.io/kube-state-metrics created
clusterrole.rbac.authorization.k8s.io/kube-state-metrics created
deployment.apps/kube-state-metrics created
serviceaccount/kube-state-metrics created
service/kube-state-metrics created
error: resource mapping not found for name: "" namespace: "" from "standard/kustomization.yaml": no matches for kind "Kustomization" in version "kustomize.config.k8s.io/v1beta1"
ensure CRDs are installed first
```

Reply:
This might break other users, see: https://github.com/kubernetes/kube-state-metrics/pull/1603#issuecomment-950156579

I would prefer not to add kustomize to this repository if there's no backwards compatible way.